src/composition/supergraph/config/lazy/mod.rs

```diff
@@ -0,0 +1,10 @@
+//! Provides objects related to lazily resolving a supergraph config.
+//!
+//! Lazy resolution is the process of taking a subgraph config and producing
+//! values that, if they contain file paths, are fully resolvable on the file system.
+
+mod subgraph;
+mod supergraph;
```

Comment:
this description is sort of hard to understand when compared with the full resolution blurb above; what's the difference between the two in practical terms? which should be used for what? a lazily resolved supergraph sounds like a local-only thing that produces something that _could_ become a fully resolved supergraph config

Reply:
'full' and 'lazy' seem like the wrong words to me; it almost feels like complete and incomplete (respectively) would fit better for what they're doing, and maybe make it clearer when/where they should be used (lazily sounds like it's fully resolvable, but under conditions that are unclear given just these descriptions and names)